packages/protocol/utils/generate_genesis/taikoL2.ts

```diff
@@ -8,6 +8,7 @@ const {
     getStorageLayout,
 } = require("@defi-wonderland/smock/dist/src/utils");
 const ARTIFACTS_PATH = path.join(__dirname, "../../out");
+
```

Comment:
@davidtaikocha Somehow my automatic git reformat will end up with a different styling with yours. Please make sure you github commit hook runs.